ui/src/components/harvest-games/HarvestGamesPage.js

```diff
@@ -0,0 +1,171 @@
+          
+          height = '90%'> {/*important so that panels do not overflow*/}
+         
+        {/* DESKTOP */}
+
+          <Tabs 
+          orientation='vertical'
+          display={['none','none','flex']}
+          borderBottom="CaptionText"
+          borderRadius="5px"
+          variant="unstyled"
+          mb="5%"
+          default={1}
+          >
+
+          <TabList border="none" width = '250px' paddingRight={"10"}>
+            <Tab 
+              borderBottom="5px solid #E2E8F0" 
+              _selected={{ borderColor: '#0628A3', textColor: "#3A6693", fontWeight: 'bold' }}>
+              <Text>
+                Leaderboard
+              </Text>
+              </Tab>
+            <Tab 
+              borderBottom="5px solid #E2E8F0"
+              _selected={{ borderColor: '#0628A3', textColor: "#3A6693", fontWeight: 'bold'}}>Pre-Games</Tab>
+          </TabList>     
+
+          <TabPanels bgColor='#FFFFFF' borderRadius={'10'} bottomPadding = "50px" >
+            <TabPanel  height = "40em">
+            <Text
+              fontSize = {{base: '20px',sm:'25px', md:'30px', lg:'40px'}}
+              textAlign = 'center'
+            >
+              Leadership BOARD: 
+              </Text>
+            </TabPanel>
+            
+            <TabPanel  height = "80em">
+            <Text
+              fontSize = {{base: '30px',sm:'30px', md:'35px', lg:'40px'}}
+              textAlign = 'center'
+            >
+               Access Pre-Game Instructions PDF <br></br>by clicking {' '}
+               <Link color='teal.500' href='https://www.youtube.com/' isExternal>
+                 here
+               </Link>
+               .
+              </Text>
+            </TabPanel>
+          </TabPanels>
+        </Tabs>
+
+        {/* MOBILE */}
+
+        <Tabs 
+          isFitted variant = 'enclosed'
+          orientation='horizontal'
+          display={['block','block','none']}
+          default={1}
+          >
+          <TabList>
+            <Tab
+              borderBottom="5px solid #E2E8F0" 
+              _selected={{ borderColor: '#0628A3', textColor: "#3A6693", fontWeight: 'bold' }}
+            >Leaderboard</Tab>
+            <Tab
+              borderBottom="5px solid #E2E8F0" 
+              _selected={{ borderColor: '#0628A3', textColor: "#3A6693", fontWeight: 'bold' }}
+            >Pre-Games</Tab>
+          </TabList>     
+
+          <TabPanels bgColor='none' borderRadius={'10'}>
+
+            <TabPanel height={{base:'30em', sm: "40em"}}> 
+              <Text
+                fontSize = {{base: '20px',sm:'25px', md:'30px', lg:'40px'}}
+                textAlign = 'center'
+              >
+                 Leadership Board:
+              </Text>
+            </TabPanel>
+            
+            <TabPanel height={{base:'30em', sm: "40em"}} >
+            <Text
+              fontSize = {{base: '25px',sm:'25px', md:'30px', lg:'40px'}}
```

Comment:
double check this font size? the access pre games instructions look bigger on this tab compared to the leaderboard